packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/CSharpGen.cs

```diff
@@ -29,7 +29,7 @@ public async Task ExecuteAsync()
             var generatedTestOutputPath = CodeModelPlugin.Instance.Configuration.TestGeneratedDirectory;
 
             GeneratedCodeWorkspace workspace = await GeneratedCodeWorkspace.Create();
-            await CodeModelPlugin.Instance.InitializeSourceInputModelAsync();
+            CodeModelPlugin.Instance.SourceInputModel = new SourceInputModel(await workspace.GetCompilationAsync());
```

Comment:
Just curious why we change the initializing here.

Reply:
just to make it simpler because this method does not have to be async any more, we could assign the constructed SourceInputModel.